.github/workflows/regression-tests.yaml

```diff
@@ -82,12 +82,29 @@ jobs:
         # (see https://github.com/solo-io/solo-projects/issues/6094)
         image-variant:
           - standard
+        version-files:
+          - label: 'min'
+            file: './.github/workflows/.env/pr-tests/min_versions.env'
+          - label: 'max'
+            file: './.github/workflows/.env/pr-tests/max_versions.env'
```

Comment:
Can we move the `kube-version` versions into the env file as well so we don't mix ways of tracking versions? I wouldn't be opposed to re-using the nightly env files, though renaming the files to be more accurate would be tricky.